web/services/management/commands/removeolddata.py

```diff
@@ -0,0 +1,17 @@
+from django.core.management.base import BaseCommand
+from services.models import ErrorReport
+from datetime import timedelta
+from django.utils import timezone
+
+
+class Command(BaseCommand):
```

Comment:
Would it be possible to provide options here? It might be useful to override the default 30 days and also have a separate option for removing everything?